### PR TITLE
[SMALLFIX] Try to keep write type consistent with understoragetype an…

### PR DESCRIPTION
**Pull request by apc999**
Try to keep write type consistent with understoragetype and tachyontype.

So let us don't define write type by default.
In case it is defined, infer understoragetype and tachyontype from write type. In code base, write type is never used.

### CODE REVIEW

File: common/src/main/java/tachyon/conf/TachyonConf.java [hunk 1]
```diff
@@ -170,6 +170,32 @@ public TachyonConf() {
...
+        mProperties.setProperty(Constants.USER_FILE_TACHYON_STORAGE_TYPE_DEFAULT, "STORE");
+        mProperties.setProperty(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT, "SYNC_PERSIST");
+      } else if (writeType.equals("THROUGH")) {
+        mProperties.setProperty(Constants.USER_FILE_TACHYON_STORAGE_TYPE_DEFAULT, "NO_STORE");
+        mProperties.setProperty(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT, "SYNC_PERSIST");
+      } else if (writeType.equals("ASYNC_THROUGH")) {
+        mProperties.setProperty(Constants.USER_FILE_TACHYON_STORAGE_TYPE_DEFAULT, "STORE");
+        mProperties.setProperty(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT, "ASYNC_PERSIST");
+      } else {
+        throw new IllegalArgumentException("Unknown property for " + Constants.USER_FILE_WRITE_TYPE_DEFAULT + " " + writeType);
```
Comment (calvinjia) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: I think this line is too long?

Comment (apc999) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: Fixed.

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**